README.md

```diff
@@ -50,7 +50,6 @@ You can modify these configurations or add new ones to the list. You can use oth
 
 For example, you can modify the `target` field to specify the simulator you want to target for iOS debugging.
 
-* **Note:** If `target` not specified we will use target from `.vscode/setting.json`.<br>
```

Comment:
You would need to add a notice that if available, arguments for running app will be read from `.vscode/settings.json` and the 'target' will affect on whether we will run app on device/emulator